code/modules/power/singularity/generator.dm

```diff
@@ -8,14 +8,16 @@
 	density = 1
 	use_power = 0
 	var/energy = 0
+	var/creation_type = /obj/singularity
 
 /obj/machinery/the_singularitygen/process()
 	var/turf/T = get_turf(src)
 	if(src.energy >= 200)
-		message_admins("A singularity has been created at [x], [y], [z] (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
-		investigate_log("A singularity has been created at [x], [y], [z]","singulo")
+		message_admins("A [creation_type.name] has been created at [x], [y], [z] (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
+		investigate_log("A [creation_type.name] has been created at [x], [y], [z]","singulo")
 
-		new /obj/singularity/(T, 50)
+ 		var/obj/singularity/S = new creation_type(T, 50)

```

Comment:
This isn't playing nice with TRAVIS, it thinks you are trying to make a new proc.

Originally, this was /obj/singularity/(T, 50) which presumably is not the same as /obj/singularity(T,50) as this line would result in.